uint8 RED=0
uint8 BLUE=1
uint8 BLINKING_GREEN=2
uint8 color
